CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # @guardian/cdk
 
+## 59.5.3
+
+**DO NOT USE**: Published by mistake.
+
```

Comment:
Add a changelog entry for 59.5.3 to let people know why it exists and to not use it.